server/enterprise/message_export/message_export_test.go

```diff
@@ -2223,21 +2556,47 @@ func readFileFromZip(t *testing.T, zipBytes []byte, filename string) string {
 	return string(contents)
 }
 
-func openZipAndReadFirstFile(t *testing.T, backend filestore.FileBackend, path string) string {
+func openZipAndReadFileNum(t *testing.T, backend filestore.FileBackend, path string, fileNum int) string {
+	zipBytes, err := backend.ReadFile(path)
+	require.NoError(t, err)
+	return readFilenumFromZip(t, zipBytes, fileNum)
+}
+
+func openZipAndReadFileStartingWith(t *testing.T, backend filestore.FileBackend, path string, startsWith string) string {
 	zipBytes, err := backend.ReadFile(path)
 	require.NoError(t, err)
-	return readFirstFileFromZip(t, zipBytes)
+
+	zipReader, err := zip.NewReader(bytes.NewReader(zipBytes), int64(len(zipBytes)))
+	require.NoError(t, err)
+
+	var names []string
+	for _, f := range zipReader.File {
+		if strings.HasPrefix(f.Name, startsWith) {
+			file, err := f.Open()
+			require.NoError(t, err)
+			contents, err := io.ReadAll(file)
+			require.NoError(t, err)
+			err = file.Close()
+			require.NoError(t, err)
+
+			return string(contents)
+		}
+		names = append(names, f.Name)
+	}
+
+	require.True(t, false, "called openZipAndReadFileStartingWith but didn't file file starting with: %s. Found: %v", startsWith, names)
```

Comment:
typo?

Reply:
Nope, if it gets there, it's an error, so this reports it.

Reply:
The `called openZipAndReadFileStartingWith but didn't file file starting with` string just read confusing. Not a big deal.